contracts/src/UniswapV3MultiQuoter.sol

```diff
@@ -280,6 +298,9 @@ contract UniswapV3MultiQuoter {
                     ? (amounts[state.amountsIndex], state.amountCalculated)
                     : (state.amountCalculated, amounts[state.amountsIndex]);
 
+                // TODO: find a better way to estimate gas. This is not an ideal way to measure gas
```

Comment:
Maybe we can track the last used tickIndex and if it's the same, then carry forward the previous gas usage number.

Reply:
I was considering returning number of ticks crossed per sample amount and having MultiQuoter return some linear formula for determining the gas usage. I think gas estimate is linearly related to the number of ticks crossed so we can do some sort of linear regression across a bunch of samples to determine what the function parameters are.